### ISSUE TITLE
Possible typo: undesirable => desirable

### ISSUE THREAD

**Issue by hawkinsw** [ThePhD/future_cxx#53]
https://github.com/ThePhD/future_cxx/blob/754a618c85cb26d84d2c5b2ceff9a50ff1ade514/papers/source/C%20-%20Consistent%2C%20Warningless%2C%20and%20Intuitive%20Initialization%20with%20%7B%7D.bs#L286

Hello!

I am a huge fan of your work and, especially, this paper. I was reading through it tonight and think that I spotted a very, very minor typo. In the line reference above you say, 

>  Note that a user can always fall back to using = { 0 } if leaving other non-overlapping values in a union is undesirable.

The way that I understand the point you are making, it would seem that undesirable there should be desirable. Again, the way I read it, you are making the point that a programmer may *want* to leave the bytes between the end of the first union element and the largest union element uninitialized for some reason. The proposal (ie, using `={}`) would negate the programmer's intention and you are saying that they could always use `= { 0 }` if they wanted the "helpful" behavior. 

I hope that this is a helpful comment and that I am not misunderstanding you. Either way, keep up the great work!

Will

**Comment by ThePhD** [comment]
Hey, this is fixed! Thanks for pointing it out.

**Comment by hawkinsw** [comment]
No, thank *you*. I sincerely appreciate all the work that you are doing!
